main.py

```diff
@@ -1,10 +1,49 @@
+from typing import Optional
+
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
+def race_create(player_race_data: dict) -> Race:
+    race_name = player_race_data["name"]
+    race, created = Race.objects.get_or_create(
+        name=race_name,
+        description=player_race_data["description"]
+    )
+    if created:
+        for skill_data in player_race_data["skills"]:
+            Skill.objects.create(
+                name=skill_data["name"],
+                bonus=skill_data["bonus"],
+                race=race
+            )
+    return race
+
+
+def guild_create(guild_data: Optional[dict] = None) -> Guild | None:
+    if guild_data is None:
+        return
+    guild_name = guild_data["name"]
+    guild, created = Guild.objects.get_or_create(
```

Comment:
The variable `created` is not used. You can use `_` instead of variable names in cases like this - then it won't be highlighted as an unused variable in the IDE